rllib/agents/dqn/r2d2_tf_policy.py

```diff
@@ -45,7 +45,8 @@ def build_r2d2_model(policy: Policy, obs_space: gym.spaces.Space,
     model = build_q_model(policy, obs_space, action_space, config)
 
     # Assert correct model type.
-    assert model.get_initial_state() != [], \
+    assert (model.get_initial_state() != [] or
```

Comment:
Attention nets don't publish init state. They use the traj. view API only.

Reply:
Can we add this comment as a comment in the code? Will that make the intention of the 2nd condition clearer here?
Thanks.

Reply:
done